packages/lit/package.json

```diff
@@ -136,14 +136,128 @@
+  },
+  "wireit": {
+    "build": {
+      "dependencies": [
+        "build:rollup",
+        "build:ts",
+        "build:ts:types"
+      ]
+    },
+    "build:ts": {
+      "#comment": "Note this also builds polyfill-support via a TypeScript project reference.",
+      "command": "tsc --build --pretty",
+      "clean": "if-file-deleted",
+      "dependencies": [
+        "../lit-element:build:ts:types",
+        "../lit-html:build:ts:types",
+        "../reactive-element:build:ts:types"
+      ],
+      "files": [
+        "src/**/*.ts",
+        "tsconfig.json",
+        "tsconfig.polyfill-support.json"
+      ],
+      "output": [
+        "development",
+        "tsconfig.tsbuildinfo",
+        "tsconfig.polyfill-support.tsbuildinfo"
+      ]
+    },
+    "build:ts:types": {
+      "command": "treemirror development . \"**/*.d.ts{,.map}\"",
+      "dependencies": [
+        "../internal-scripts:build",
+        "build:ts"
+      ],
+      "files": [],
+      "output": [
+        "*.d.ts{,.map}",
+        "decorators/*.d.ts{,.map}",
+        "directives/*.d.ts{,.map}"
+      ]
+    },
+    "build:rollup": {
+      "command": "rollup -c",
+      "dependencies": [
+        "build:ts",
+        "../lit-element:build:rollup"
+      ],
+      "files": [
+        "rollup.config.js",
+        "../../rollup-common.js"
+      ],
+      "output": [
+        "async-directive.js{,.map}",
+        "decorators.js{,.map}",
+        "directive-helpers.js{,.map}",
+        "directive.js{,.map}",
+        "html.js{,.map}",
+        "experimental-hydrate-support.js{,.map}",
+        "experimental-hydrate.js{,.map}",
+        "index.js{,.map}",
+        "polyfill-support.js{,.map}",
+        "static-html.js{,.map}",
+        "decorators/*.js{,.map}",
+        "directives/*.js{,.map}"
+      ]
+    },
+    "checksize": {
+      "command": "rollup -c --environment=CHECKSIZE",
+      "dependencies": [
+        "build:ts"
+      ],
+      "files": [
+        "rollup.config.js",
+        "../../rollup-common.js"
+      ],
+      "output": []
+    },
+    "test": {
+      "dependencies": [
+        "test:dev",
+        "test:prod"
+      ]
+    },
+    "test:dev": {
+      "#comment": "TODO(aomarks) There is only one lit html test, and it doesn't work",
+      "#comment": "TODO(aomarks) Factor out a file-only target in tests/ to DRY these config files into a single place",
+      "command": "cd ../tests && MODE=dev npx wtr \"../lit/development/**/*_test.(js|html)\"",
+      "dependencies": [
+        "build:ts"
+      ],
+      "files": [
+        "../tests/web-test-runner.config.js",
+        "../tests/wtr-config.js",
+        "../tests/rollup-resolve-remap.js"
+      ],
+      "output": []
+    },
+    "test:prod": {
+      "command": "cd ../tests && MODE=prod npx wtr \"../lit/development/**/*_test.(js|html)\"",
+      "dependencies": [
+        "../lit-element:build:rollup",
+        "../lit-html:build:rollup",
+        "../reactive-element:build:rollup",
```

Comment:
We did need it, because `reactive-element/polyfill-support.js` bundles in `lit-html/polyfill.support.js` and `lit-element/polyfill-support.js`, and `lit/polyfill-support.js` bundles in `lit-element.polyfill-support.js`.

It did this with an `import`, and letting Rollup do its default module resolution, which is to the `production` build, which is the output of Rollup. Hence the dependencies.

But now I've fixed that by changing the export condition to `development`, and confirmed that the `polyfill-support.js` files are identical before and after. Now each Rollup step only depends on the `build:ts` of its dependencies.